### PR TITLE
feat: add alt + shift modifiers

### LINKED ISSUE #33: `Alt` + `Shift` modifiers
VHS should support `Alt` and `Shift` modifiers for keypress:

```
Ctrl+Shift+O
Ctrl+Alt+L
Ctrl+Alt+Shift+P
Ctrl+Shift+Alt+C
```

### PR DESCRIPTION
**Pull request by elmarsan**
This pull request aims to add support for `Alt` and `Shift` modifiers.

Closes #33

### CODE REVIEW

**Comment by maaslalani** [comment]
Hey @elmarsan, this PR looks great.

1 suggestion is can we change the syntax from:

```
Ctrl+N<char|modifier>
```

to:

```
Ctrl[+Alt][+Shift]+<char>
```

File: parser.go [hunk 1]
```diff
@@ -130,21 +130,49 @@ func (p *Parser) parseTime() string {
...
+		peek := p.peek
+
+		// Get key from keywords and check if it's a valid modifier
+		if k, ok := keywords[peek.Literal]; ok {
+			p.nextToken()
+			if IsModifier(k) {
+				args = append(args, peek.Literal)
+			} else {
+				msg := fmt.Sprintf("Keyword : '%s' can not be used as control argument", p.cur.Literal)
+				p.errors = append(p.errors, NewError(p.cur, msg))
```
Comment (maaslalani) [inline parser.go:153]:
```suggestion
				p.errors = append(p.errors, NewError(p.cur, "not a valid modifier"))
```

**Comment by elmarsan** [comment]
@maaslalani it should be ready 😀

**Comment by maaslalani** [comment]
Awesome @elmarsan, going to get this one in today for sure after testing it out.

**Comment by maaslalani** [comment]
Amazing work @elmarsan! This is fantastic.

**Pull request merged**